internal/cli/config/set.go

```diff
@@ -53,9 +53,6 @@ func (opts *SetOpts) Run() error {
 }
 
 func SetBuilder() *cobra.Command {
-	opts := &SetOpts{
-		store: config.Default(),
-	}
```

Comment:
Because this was initialised before `Load` might be called, this would always be `default`